.swiftlint.yml

```diff
@@ -2,6 +2,12 @@ opt_in_rules:
 - empty_count
 - empty_string
 
+disabled_rules:
+- nesting
+- identifier_name
```

Comment:
If `identifier_name` is because of `id` usage then we can make a custom config to remove it from the excluded names but then still benefit from the other rules.

Reply:
fixed